src/components/Questions/QuestionModal.vue

```diff
@@ -146,9 +166,9 @@ export default defineComponent({
     }
 
     function navigateToQuestion(questionIndex: number) {
-      if (!isQuestionFetched(questionIndex)) {
-        context.emit("fetch-question-bucket", questionIndex)
-      }
+      context.emit("fetch-question-bucket", questionIndex)
+      state.reRenderKey = !state.reRenderKey
+      resetState()
```

Comment:
any reason why this was added?

Reply:
i dont remember the error exactly but we rerender component whenever we click previousQuestion, or nextQuestion, etc. So we should be rerendering components when we navigate to new question via palette too na?